proto/http.c

```diff
@@ -301,14 +301,61 @@ static int http_parse(struct wsgi_request *wsgi_req, char *watermark) {
 		}
 	}
 	else {
-		// TODO add ipv6 support
-		memset(ip, 0, INET_ADDRSTRLEN+1);
-		if (inet_ntop(AF_INET, (void *) &http_sin->sin_addr.s_addr, ip, INET_ADDRSTRLEN)) {
-			wsgi_req->len += proto_base_add_uwsgi_var(wsgi_req, "REMOTE_ADDR", 11, ip, strlen(ip));
-		}
-		else {
-			uwsgi_error("inet_ntop()");
-			return -1;
+		// TODO log something useful for AF_UNIX sockets
+		switch (http_sa->sa_family) {
+			case AF_INET6:
```

Comment:
Removing the brackets changes the scoping semantics (it means that I need to effectively hoist the variables defined in these sections to the top level). I can do that, but I'm not sure it's the desired goal.

As far as alignment, I was trying to match the existing `case/switch` usage in this file (e.g., `uwsgi_is_full_http` on line 750). If your preference is to align `case` with `switch`, I'll do so.